pom.xml

```diff
@@ -2,9 +2,85 @@
 <project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
+
   <groupId>io.strimzi</groupId>
   <artifactId>strimzi-drain-cleaner</artifactId>
   <version>1.2.0-SNAPSHOT</version>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <name>Strimzi Drain Cleaner</name>
+  <description>Utility to help with draining nodes running Strimzi-managed Apache Kafka
+    or Apache ZooKeeper pods. </description>
+  <url>https://strimzi.io/</url>
+
+  <scm>
+    <connection>scm:git:git://github.com/strimzi/drain-cleaner.git</connection>
+    <developerConnection>scm:git:ssh://github.com:strimzi/drain-cleaner.git</developerConnection>
+    <url>https://github.com/strimzi/drain-cleaner</url>
+  </scm>
+
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/strimzi/drain-cleaner/issues</url>
+  </issueManagement>
+
+  <developers>
```

Comment:
Sam Hawker is missing from the list below

Reply:
Now added.